### DISCUSSION TITLE
I was marked as spam, the reason is a recursive in github actions

### DISCUSSION THREAD

**Discussion by cuong2905say** [community/community#144424] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Pull Requests

### Body

Hi community, I created a project, which has a workflow file. Nothing until there is a recursion loop in there:
```
on:
   push:
      branches:
         - master      
```

In the jobs, I created pull request with another branch, and merge it to master. The workflows triggered and so many workflow run in a time. Github recognize me as a spam and make me invisible from public. I created a ticket but a week passed. I want to know the ratio my account back, and If it cannot be back, will Github notice me about the reason or simply they will pass with no notification to me?. Please help, I need me account to submit my homework.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @cuong2905say, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
